feed2discord.py

```diff
@@ -623,6 +624,10 @@ async def background_check_feed(feed, asyncioloop):
             # pull data out of the http response
             logger.info(feed + ":reading http response")
             http_data = await http_response.read()
+            ##fuck you acast
```

Comment:
I'd prefer that kind of behavior be conditional on a per-feed setting that defaults to false... Or at least doing a smarter check on exactly the stuff to replace, since that could catch stuff inside text itself...